pymedphys/_experimental/pseudonymisation/__init__.py

```diff
@@ -115,3 +116,22 @@ def anonymise_with_pseudo_cli(args):
         raise FileNotFoundError(
             "No file or directory was found at the supplied input path."
         )
+
+
+def is_valid_strategy_for_keywords(
+    identifying_keywords=None, replacement_strategy=None
+):
+    test_keywords = identifying_keywords
+    if test_keywords is None:
+        test_keywords = get_default_pseudonymisation_keywords()
+    test_strategy = replacement_strategy
+    if test_strategy is None:
+        test_strategy = strategy.pseudonymisation_dispatch
```

Comment:
I think here it's equivalent to write:

```python
    if identifying_keywords is None:
        identifying_keywords = get_default_pseudonymisation_keywords()
    if replacement_strategy is None:
        replacement_strategy = strategy.pseudonymisation_dispatch
```

Then just use `identifying_keywords` and `replacement_strategy` below.